shiny/Data-Ingest/ui_files/input_record_ui.R

```diff
@@ -118,6 +120,7 @@ fluidRow(title = "New Input",
          height = '150px'
        ),
        actionButton("createFormatRecord", label = "Create Format Record"),
+       p("* Denotes a Required Field"),
```

Comment:
Not completely following the logic of which fields are listed as required, as many of the starred fields seem optional unless other choices are made in specific combinations. More generally, some screenshots would be nice to better see what this actually looks like

Reply:
Ok! Noted. It would be great to discuss what fields are required for each input. I put in the asterisks assuming that some of these fields would be required (based on my best judgement). Beyond that, I'm not sure I understand the logic either. And OF COURSE! I figured you would be able to view the changes once they were merged, but good point about the screen shots! I'll post those now.

Reply:
Also just noticed that this is the two thousandth pull request YAY!